clients/client-py/setup.py

```diff
@@ -96,8 +90,10 @@ def run_tests(self):
         },
         cmdclass={'test': Tox},
         zip_safe=False,
-        classifiers=['Programming Language :: Python :: 2.7',
-                     'Programming Language :: Python :: 3.5',
-                     'Programming Language :: Python :: 3.6',
-                     'Programming Language :: Python :: 3.7'],
+        classifiers=[
```

Comment:
Two questions here:
- is this client Python 3.10 compatible? (or is that out of scope to account for in this PR?)
- I noticed in other places that we removed references to Python 3.6 (which is also at EOL). Should we remove the classifier here?

Reply:
I think that's out of scope.  We typically haven't dropped support for old versions until they become a nuisance (as 2.7 did recently), so I'd suggest keeping 3.6 for now.  Adding 3.10 is hopefully just a matter of adding a test task for it.

The removals of `; python_version>="3.6"` were removals of a filter to only install those deps on Python-3, not to drop Python-3.6 support.